Change "not activated" to "not ready"

jgustie opened the pull request.

jgustie committed:
Change "not activated" to "not ready"

bradbeam commented on redskyapi/experiments/v1alpha1/http.go:
```diff
@@ -426,7 +426,7 @@ func newError(t ErrorType, resp *http.Response, body []byte) error {
 			err.Type = ErrUnauthorized
 			if err.Message == "" {
-				err.Message = "account is not activated"
+				err.Message = "your account is not ready, please check https://app.carbonrelay.io/ for more details"
```
do we need to flex on the url? ( diff between prod/dev ? )

jgustie replied:
It might be nice to change the `login` command to use `https://staging.redskyops.dev/api/auth_success` and `https://app.carbonrelay.dev/` for non-release builds, but in general the CLI is abstracted from the backend deployment environments.

bradbeam commented on redskyctl/internal/commands/login/login.go:
```diff
@@ -288,9 +288,9 @@ func (o *Options) generateCallbackResponse(w http.ResponseWriter, r *http.Reques
 	default:
 		// TODO Better detection of this error
-		if status == http.StatusInternalServerError && err != nil && err.Error() == "account is not activated" {
+		if status == http.StatusInternalServerError && err != nil && strings.HasPrefix(err.Error(), "your account is not ready") {
```
Should we make an actual error type for this, `api.errAccountNotReady` ?

jgustie committed:
Change the "not activated" code to "missing namespace claim"

jgustie committed:
Remove URL from API generated error message

In theory you should never get past login in this state, so it is only important to have the more friendly verbose error there
